renderdoc/driver/vulkan/vk_replay.cpp

```diff
@@ -1434,19 +1434,24 @@ void VulkanReplay::SavePipelineState(uint32_t eventId)
       default: break;
     }
 
-    ret.rasterizer.lineStippleFactor = state.stippleFactor;
-    ret.rasterizer.lineStipplePattern = state.stipplePattern;
+    ret.rasterizer.lineStippleFactor = 0;    // stippled line disable
```

Comment:
You should set the pattern to 0 here also, to ensure it's always set to something up to date. The state is updated in-place to avoid copying large structs so this would leave the pattern member with stale data.